src/4_initialization.js

```diff
@@ -23,12 +23,13 @@ if (window['branch'] && window['branch']['_q']) {
 
 // AMD
 if (typeof define === 'function' && define.amd) {
-	define('branch', function() { return branch_instance; });
+	define([ 'branch' ], function() { return branch_instance; });
```

Comment:
This would create an anonymous module that requires 'branch', instead of creating a 'branch' module.